README.md

```diff
@@ -1,19 +1,83 @@
-# BSOC
+<div align="center">
 
-## Project setup
-```
-npm install
-```
+# 👨‍💻BSoC Website
+####  Hello developers✨!!..welcome to Bitbyte Summer of Code.. a wonderful chance for dedicated coders. 
+</div>
 
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
+This website seeks to connect you with open source, free software, and tech-related initiatives where you may study and develop code. The projects offer mentors who serve as guides throughout the whole process, from getting to know the community to contributing code. 
 
-### Compiles and minifies for production
-```
-npm run build
+## 🗒️Features
+- Supports you to start taking part in open source development
+- The leaderboard lists the top performers,which gets updated every day 
+- Provides you with the chance to work with different people on the same project
+- Real-life experience about many fields 
+- Contains FAQs to resolve your queries
+
+## ⚙️Tech Stack
+### Frontend:
+- HTML5 
+- CSS
+- ReactJS
+- VUE
+### Backend:
+- Firebase
```

Comment:
ReactJS is not used in this, it only uses VueJs and dont mention it as backend and frontend just write all the technologies used.